pallets/gear/src/lib.rs

```diff
@@ -1579,7 +1579,7 @@ pub mod pallet {
 
         /// Process message queue
         #[pallet::call_index(6)]
-        #[pallet::weight((Weight::zero(), DispatchClass::Mandatory))]
+        #[pallet::weight((<T as frame_system::Config>::BlockWeights::get().max_block, DispatchClass::Mandatory))]
```

Comment:
Need to deal with exhausting block resources. Consider use weight as max_block - used. It will result in really used = used / 2?

Reply:
Block can't exhaust cuz it's Mandatory `DispatchClass`